src/main/java/com/iexec/worker/tee/post/compute/web2/UploaderService.java

```diff
@@ -64,17 +66,22 @@ public static String uploadToIpfsWithIexecProxy(String taskId, String baseUrl, S
 
         HttpEntity<ResultModel> request = new HttpEntity<>(resultModel, headers);
 
-        ResponseEntity<String> response = new RestTemplate().postForEntity(baseUrl, request, String.class);
-
-        if (response.getStatusCode().is2xxSuccessful()) {
-            return response.getBody();
+        HttpStatus statusCode = null;
+        try {
+            ResponseEntity<String> response = new RestTemplate().postForEntity(baseUrl, request, String.class);
+            statusCode = response.getStatusCode();
+
+            if (statusCode.is2xxSuccessful()) {
```

Comment:
status != null && is2xxSuccessful
(just in case)

Reply:
`statusCode` could never be null here. When adding this check, IntellIj triggers a warning. If it's OK for you, let's NOT add this check :)

Reply:
Yes, thank you for checking!